src/ontology/mondo-ingest.Makefile

```diff
@@ -199,6 +199,14 @@ $(REPORTDIR)/%_mapping_status.tsv $(REPORTDIR)/%_unmapped_terms.tsv: $(REPORTDIR
 	--outpath-simple $(REPORTDIR)/$*_unmapped_terms.tsv \
 	--outpath-full $(REPORTDIR)/$*_mapping_status.tsv
 
+components/%-unmapped.owl: components/%.owl reports/%_unmapped_terms.tsv
+	cut -f 1 reports/$*_unmapped_terms.tsv | tail -n +2 > reports/$*_unmapped_terms.txt
+	$(ROBOT) filter -i components/$*.owl -T reports/$*_unmapped_terms.tsv -o $@
+	rm reports/$*_unmapped_terms.txt
+
+.PHONY: recreate-unmapped-components
+recreate-unmapped-components: $(patsubst %, components/%-unmapped.owl, $(ALL_COMPONENT_IDS))
```

Comment:
@matentzn What you had in the GitHub issue for this goal was slightly off. I tried a variation that didn't work, then asked ChatGPT and the robot overlord succeeded!

Reply:
I am sure :P Great job here!